simapp/app.go

```diff
@@ -231,7 +233,7 @@ func NewSimApp(
 	runtimeApp := appBuilder.Build(logger, db, traceStore, baseAppOptions...)
 
 	keys := sdk.NewKVStoreKeys(
-		authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
```

Comment:
Bit of an issue here surfaced in tests.
`authTypes.Store = "acc"` (not `"auth"`), but [runtime.ProvideKVStoreKey](https://github.com/cosmos/cosmos-sdk/blob/8eaff8fadd408762fa5a3c5a40969bda4548057e/runtime/module.go#L104) makes the assumption that `storeKey = moduleName` (`"auth"`)

Reply:
Hmm... this could be an issue when migrating existing stores. One option is to give the module the name acc in the app config. In general I think the existing practice of hard coding these names is wrong, but we need to cope with how things are currently done too... we'll need to think about this a bit

Reply:
Two options I can think of:
1. Let modules give a store key name, not a huge fan of this
2. Let users specify an alternate store key name for a module in the runtime config 

Sort of leaning towards 2

Reply:
I favor (2) as well (key overrides)

Reply:
implementation of (2) pushed for review in https://github.com/cosmos/cosmos-sdk/pull/12019/commits/90759acc90587e00e2bb4bc64c70909f43b5562e